### PR TITLE
deflake integration test

### PR DESCRIPTION
**Pull request by deads2k**
finalize subresource does a conditional update which can conflict with a controller adding UIDs.

/assign @smarterclayton

### CODE REVIEW

File: test/integration/namespace_lifecycle_admission_test.go [hunk 1]
```diff
@@ -63,13 +64,20 @@ func TestNamespaceLifecycleAdmission(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	ns.Spec.Finalizers = append(ns.Spec.Finalizers, projectapi.FinalizerOrigin)
-	t.Log(spew.Sdump(ns))
-	afterUpdate, err := clusterAdminKubeClientset.Core().Namespaces().Finalize(ns)
+	err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		ns, err = clusterAdminKubeClientset.Core().Namespaces().Get("test", metav1.GetOptions{})
```
Comment (deads2k) [inline test/integration/namespace_lifecycle_admission_test.go]: do a get to make sure you get current objects on retries.

**Comment by smarterclayton** [comment]
Will you remove the spew before merge, or just merging to get the spew?

**Comment by smarterclayton** [comment]
Batch passed, but didn't get merged - manual merge

**Pull request merged**